src/App.svelte

```diff
@@ -1,9 +1,7 @@
 <script>
-  import svelteLogo from './assets/svelte.svg';
-  import { Badge } from 'flowbite-svelte';
+  import Header from './components/Header.svelte';
 </script>
 
 <main>
-  <Badge>App</Badge>
-  <img src={svelteLogo} alt="Svelte Logo" />
+  <Header />
```

Comment:
Maybe some styles to set up position for header is required too?

Reply:
> Maybe some styles to set up position for header is required too?

Not sure that I get you. What king of styles do you mean to I add?

Reply:
Done, sticky position added